packages/twig-new/src/components/tags/tag.component.yml

```diff
@@ -8,20 +8,17 @@ tag:
       type: text
       label: Content
       description: The tag text content.
-      preview:
-        faker: lorem.word
+      preview: content
```

Comment:
This is value that is supplied as the default to the field right? Maybe Tag is better text for here